src/mlpack/core/util/sfinae_utility.hpp

```diff
@@ -182,6 +182,34 @@ struct NAME                                                                    \
       WithGreaterOrEqualNumberOfAdditionalArgs<MinN>::value;                   \
 };
 
+/**
+ * Constructs a template structure, which will define a boolean static variable
+ * ,to true if the passed template parameter, has a member function with the
+ * specified name. The check does not care about the signature or the function
+ * parameters
+ *
+ * @param FUNC the name of the function, whose existence is to be detected
```

Comment:
Another pedantic style issue; do you mind to use the correct punctuation here, e.g. end with a ``.``.

Reply:
Yes. Sorry that was a bad miss.